docs/index.md

```diff
@@ -157,6 +596,11 @@ Please note that this package is currently under development. While you can
 download and test the functionalities that are already present, please be aware
 that syntax and functionality may change before our preliminary release.
 
+For those looking to get a better grasp of the Generalized Linear Model, we recommend checking out the 
```

Comment:
I don't think this belongs here. Not sure where it belongs though. Maybe in the background readme?